authors/apps/comments/views.py

```diff
@@ -86,32 +83,34 @@ def get(self, request, pk):
         serialized_data = self.serializer_class(self.comment)
         return Response({"comment": serialized_data.data,
                          "message": (
-                    "Successfully returned details of comment: {}".format(pk)
-                         )},
-                        status=status.HTTP_200_OK)
+                             "Successfully returned details of comment: {}".
+                             format(pk)
+                         )}, status=status.HTTP_200_OK)
 
     def patch(self, request, pk):
         self.get_required_objects(request, pk)
         self.get_author_profile(request)
         if not self.is_author:
             return Response({
-                    "response": "You can only edit a comment you authored",
+                "response": "You can only edit a comment you authored",
                             "status_message": "Failed: Access denied"},
                             status=status.HTTP_403_FORBIDDEN)
         serializer = self.serializer_class(
             self.comment, self.data, partial=True)
         serializer.is_valid(raise_exception=True)
-        serializer.save()
+        if self.comment.body != self.data['body']:
+            serializer.save()
         return Response({"comment": serializer.data,
-                    "message": "Successfully edited comment: {}".format(pk)},
+                         "message": "Successfully edited comment: {}".
+                         format(pk)},
                         status=status.HTTP_201_CREATED)
 
     def delete(self, request, pk):
         self.get_required_objects(request, pk)
         self.get_author_profile(request)
         if not self.is_author:
             return Response({
-                    "response": "You can only delete a comment you authored",
+                "response": "You can only delete a comment you authored",
```

Comment:
I would rather stick with one way to return messages to the user. You are introducing response again. Previous comment.

Reply:
Working on that.

Reply:
@missvicki, there is actually a message below the response which states failure of the request. Look closely. Unless you are suggesting that the response should be removed altogether.